### DISCUSSION TITLE
后台神秘流量相关

### DISCUSSION THREAD

**Discussion by siyue2016** [vernesong/OpenClash#3859] [Q&A]
![image](https://github.com/vernesong/OpenClash/assets/126215802/f4407d7e-3a94-49fd-b1ec-c40a0ce5010c)
每时每刻都在上传
查了下可能是没打开 仅内网访问 功能导致被别人白嫖了上传流量20G 
但是打开功能后台日志报
"设置仅允许内网功能失败，无法获取 IPv4 的 WAN 接口名称，请确保防火墙设置中 IPv4 WAN 区域的名称为 wan，如设备无 WAN 口请忽略此提示"

我的设备是旁路由 主路由DHCP
旁路由接口
![image](https://github.com/vernesong/OpenClash/assets/126215802/33e3d684-759f-4e45-95b5-a507bb5da0d1)

小白请问下该怎么设置
或者说怎么避免这种偷上传流量的情况

**Comment by Plutonium141** [comment]
主路由把旁路由暴露在公网上了吧，用dmz了？

**Comment by vernesong** [comment]
![image](https://github.com/vernesong/OpenClash/assets/42875168/749e35b3-cee2-41e6-9e8d-e04178c1ab40)
防火墙名字改一下
